packages/astro/src/@types/astro.ts

```diff
@@ -419,22 +419,13 @@ export interface AstroUserConfig {
 	 * @name base
 	 * @type {string}
 	 * @description
-	 * The base path you're deploying to. Astro will match this pathname during development so that your development experience matches your build environment as closely as possible. In the example below, `astro dev` will start your server at `/docs`.
+	 * The base path to deploy to. Astro will build your pages and assets using this path as the root. Currently, this has no effect during development. 
 	 *
 	 * ```js
 	 * {
 	 *   base: '/docs'
 	 * }
 	 * ```
-	 *
-	 * When using this option, you should mind that all of your imports will be affected. In this example, all of the imports including static resources and codes should add a prefix `/docs/`.
-	 *
-	 * For example, if you want to use a image in your Astro component, you need to change it from '/someimg.png' into '/docs/someimg.png'.
-	 *
-	 * ```astro
-	 * <!-- <img src="/someimg.png"> is not correct. -->
-	 * <img src="/docs/someimg.png">
-	 * ```
```

Comment:
Curious why this part was deleted? This is still true, if a bit misphrased here.

Here's my suggestion:

> When using this option, be mindful that asset references in your project are not automatically transformed. In this example, all assets in the `public/` directory and any non-relative links should be manually prefixed with `/docs/`.
>
> ```diff
> <!-- Reference to `public/hero.png` file -->
> - <img src="/hero.png">
> + <img src="/docs/hero.png">
>
> <!-- Link to page generated by `src/pages/feature.astro` -->
> - <a href="/feature">Next</a>
> + <a href="/docs/feature">Next</a>
> ```

Reply:
If that's the case, that's a regression that needs to be fixed right away!

Reply:
I’d love to see this PR merged ASAP as we’re currently blocked on merging `@types` updates into docs because the current version (the stuff Dan deleted above) is gobbledygook. Could we settle on the current state of this PR and agree to add back this section as part of the #4988 fix?

Reply:
+1, we can get this in as-is and come back to it when #4988 is resolved